u8timeseries/timeseries.py

```diff
@@ -73,48 +71,30 @@ def __init__(self,
         # The actual values
         self._values: np.ndarray = self._series.values
 
-        # Handle confidence intervals:
-        self._confidence_lo = None
-        self._confidence_hi = None
-        if confidence_lo is not None:
-            self._confidence_lo = confidence_lo.sort_index()
-            raise_if_not(len(self._confidence_lo) == len(self._series),
-                         'Lower confidence interval must have same size as the main time series.', logger)
-            raise_if_not((self._confidence_lo.index == self._series.index).all(),
-                         'Lower confidence interval and main series must have the same time index.', logger)
-        if confidence_hi is not None:
-            self._confidence_hi = confidence_hi.sort_index()
-            raise_if_not(len(self._confidence_hi) == len(self._series),
-                         'Upper confidence interval must have same size as the main time series.', logger)
-            raise_if_not((self._confidence_hi.index == self._series.index).all(),
-                         'Upper confidence interval and main series must have the same time index.', logger)
-
-    def pd_series(self) -> pd.Series:
+    def _assert_univariate(self):
         """
-        Returns
-        -------
-        pandas.Series
-            A copy of the Pandas Series underlying this time series
+        Raises an error if the current TimeSeries instance is not univariate.
         """
-        return self._series.copy()
+        raise_if_not(self._series.shape[1] == 1, 'Only univariate TimeSeries instances support this method', logger)
```

Comment:
This one will raise a `ValueError`. Depending on the context (e.g. when called from `pd_series()`), it might not be the best error to raise, as we are not in a case where the provided function arguments are bad. Could you raise better errors that reflect the context? Maybe `AssertionError` is already better.

Reply:
done